pkg/services/stats/models.go

```diff
@@ -101,6 +104,7 @@ type AdminStats struct {
 	DailyActiveViewers  int64 `json:"dailyActiveViewers"`
 	DailyActiveSessions int64 `json:"dailyActiveSessions"`
 	MonthlyActiveUsers  int64 `json:"monthlyActiveUsers"`
+	AnonymousStats
```

Comment:
@Jguer i am pretty sure that this becomes a nil pointer, if this does not exists

Reply:
No, in this case you're using struct embedding, which means they become 'fields' of the Admin stats initialized to 0, which is fine